graylog2-server/src/main/java/org/graylog/events/processor/aggregation/AggregationEventProcessor.java

```diff
@@ -131,6 +132,11 @@ public void createEvents(EventFactory eventFactory, EventProcessorParameters pro
                 LOG.error(msg);
                 throw new EventProcessorPreconditionException(msg, eventDefinition, e);
             }
+        } catch (ElasticsearchException e) {
+            final String msg = String.format(Locale.ROOT, "Couldn't run aggregation <%s/%s> because of search error: %s",
+                    eventDefinition.title(), eventDefinition.id(), e.getMessage());
+            LOG.error(msg);
+            throw new EventProcessorPreconditionException(msg, eventDefinition.id(), eventDefinition, e);
```

Comment:
```suggestion
            throw new EventProcessorPreconditionException(msg, eventDefinition, e);
```
just a nitpick. but we haven't used the other constructor yet. And this does the same, while being shorter